dapps/profile.vue/src/components/wallet/wallet.ts

```diff
@@ -69,8 +69,8 @@ export default class WalletComponent extends mixins(EvanComponent) {
     }));
   }
 
-  async loadBalance(): Promise<void> {
-    const balance = await core.getBalance(this.getRuntime().activeAccount);
+  async loadBalance() {
+    const balance = await bccHelper.getBalance((this as any).getRuntime().activeAccount);
```

Comment:
passing as any shouldn't be necessary in EvanComponent context anymore.
```suggestion
    const balance = await bccHelper.getBalance(this.getRuntime().activeAccount);
```